app/models/local_authority.rb

```diff
@@ -85,6 +85,12 @@ def applicants_url
     end
   end
 
+  def public_register_url
```

Comment:
Would it make sense to call this `public_register_base_url` perhaps, to make the difference with `Consultation#public_register_url` clearer?

Reply:
good suggestion and it makes sense!